raspadorlegislativo/spiders/senado.py

```diff
@@ -29,11 +29,16 @@ class SenadoSpider(BillSpider):
         )
     }
 
+    @staticmethod
+    def date_parameters(start_date):
+        formatted_start_date = start_date.isoformat().replace('-', '')
+        return [formatted_start_date] + [f"{year}0101" for year in range(start_date.year + 1, date.today().year + 1)]
```

Comment:
Duas sugestões:
1. [Não precisamos de uma lista](https://cuducos.me/2018/06/13/tuplas-e-listas-no-python.html), podemos ter um [gerador](https://cuducos.me/2017/08/11/geradores-do-python-o-que-sao-e-quais-problemas-resolvem.html)
2. Para manipular datas, é melhor e mais seguro usar o objeto `date` do que manipular _strings_

```suggestion
        yield start_date
        yield from (date(year, 1, 1) for year in range(start_date.year + 1, date.today().year + 1))
```